Thunder/server/stream_routes.py

```
@@ -215,9 +215,9 @@ async def stream_generator():
             raise web.HTTPInternalServerError(text="Server error") from e
         
     except (InvalidHash, FileNotFound) as e:
-        logger.debug(f"Client error: {type(e).__name__}")
+        logger.debug(f"Client error: {type(e).__name__}", exc_info=True)
         raise web.HTTPNotFound(text="Resource not found")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)